lang/deutsche.yml

```diff
@@ -0,0 +1,19 @@
+next_move: Was ist Ihr nächster Schritt
+which_game: Welches Spiel wollen Sie?
+not_free: Diese Position ist nicht frei
+winner: der Gewinner ist
+integer: der Wert muss eine ganze Zahl sein
+computer_move: der Computer wird seine nächsten Zug spielen
+tie: es ist eine Krawatte!
+between: der Wert sollte zwischen sein
```

Comment:
the sentence structure doesn't quite work.. you'd have to say "der Wert sollte zwischen <number> und <number> sein"